secure/handler/authorizationHandler.go

```diff
@@ -7,9 +7,13 @@ import (
 
 	"github.com/Comcast/webpa-common/logging"
 	"github.com/Comcast/webpa-common/secure"
+	"github.com/SermoDigital/jose/jws"
 	"github.com/go-kit/kit/log"
 )
 
+//satClientIDKey is the key to set/get sat client IDs using contexts.
+var satClientIDKey = struct{}{}
```

Comment:
You had this right before, using a custom type:

`type contextKey struct{}`

The reason you don't want to use `struct{}{}` here as a global variable is that it can collide with other package keys.  A custom type, using `struct{}`, is canonicalized by go and is distinct from any other type.